plantdescriptionengine/src/main/java/eu/arrowhead/core/plantdescriptionengine/providedservices/pde_monitor/routehandlers/DtoUtils.java

```diff
@@ -0,0 +1,175 @@
+package eu.arrowhead.core.plantdescriptionengine.providedservices.pde_monitor.routehandlers;
+
+import eu.arrowhead.core.plantdescriptionengine.MonitorInfo;
+import eu.arrowhead.core.plantdescriptionengine.providedservices.pde_mgmt.dto.Connection;
+import eu.arrowhead.core.plantdescriptionengine.providedservices.pde_mgmt.dto.PdeSystem;
+import eu.arrowhead.core.plantdescriptionengine.providedservices.pde_mgmt.dto.PlantDescriptionEntry;
+import eu.arrowhead.core.plantdescriptionengine.providedservices.pde_mgmt.dto.Port;
+import eu.arrowhead.core.plantdescriptionengine.providedservices.pde_monitor.dto.ConnectionDto;
+import eu.arrowhead.core.plantdescriptionengine.providedservices.pde_monitor.dto.MonitorPlantDescriptionEntryDto;
+import eu.arrowhead.core.plantdescriptionengine.providedservices.pde_monitor.dto.PortEntryDto;
+import eu.arrowhead.core.plantdescriptionengine.providedservices.pde_monitor.dto.SystemEntryDto;
+import eu.arrowhead.core.plantdescriptionengine.providedservices.pde_monitor.dto.SystemPortDto;
+import eu.arrowhead.core.plantdescriptionengine.utils.Metadata;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+
+public final class DtoUtils {
+
+    private static final Logger logger = LoggerFactory.getLogger(DtoUtils.class);
+
+    private DtoUtils() {
+    }
+
+    /**
+     * Converts the provided list of {@link eu.arrowhead.core.plantdescriptionengine.providedservices.pde_mgmt.dto.Connection}
+     * to a list of {@link eu.arrowhead.core.plantdescriptionengine.providedservices.pde_monitor.dto.Connection}
+     * objects.
+     *
+     * @param connections A list of connections adhering to the mgmt package
+     *                    format.
+     * @return A list of connections adhering to the monitor package format.
+     */
+    private static List<ConnectionDto> mgmtToMonitor(final List<? extends Connection> connections) {
+        final List<ConnectionDto> result = new ArrayList<>();
```

Comment:
initial size should be connections.size()

Reply:
Fixed